### DISCUSSION TITLE
Laravel Nova accessibility & screen readers support

### DISCUSSION THREAD

**Discussion by Otienoh** [laravel/nova-issues#4852] [Ideas & Feature Requests]
This being an admin panel product, is there any effort made to make this product accessible to the visually impaired users?

What are the steps one can take to make it more accessible to these users?

Laravel Nova accessibility & screen readers support

**Comment by marklabrecque-ab** [comment]
I am also interested in this type of support. I'd be open to contributing back some solutions here if the Nova team was open to that.
